vignettes/trap_effort.Rmd

```diff
@@ -305,17 +250,75 @@ usethis::use_data(weekly_hours_fished, overwrite = TRUE)
+The following plots provide visuals to show weekly hours fished for a few select stream and calendar year. 
+Note that for the combined plots this is for all years of data so there are multiple datapoints for a given week due to multiple years being represented and multiple subsites.
 
 ```{r, echo = FALSE}
 weekly_hours_fished %>% 
-  ggplot(aes(x = week, y = hours_fished, color = factor(site))) +
+  filter(stream == "battle creek", year == 2022) |> 
+  ggplot(aes(x = week, y = hours_fished, color = factor(subsite))) +
   geom_point() + 
-  facet_wrap(~stream, scales = "free")+ 
+  facet_wrap(~year, scales = "free")+ 
   scale_color_manual(values = colors_full) +
   theme_minimal() +
       labs(x = "Week", y = "Hours Fished", 
-       title = "Hours Fished All Methods")
+           color = "",
+       title = "Batte Creek Hours Fished")
+
+weekly_hours_fished %>% 
+  filter(stream == "butte creek", year == 2022) |> 
+  ggplot(aes(x = week, y = hours_fished, color = factor(subsite))) +
+  geom_point() + 
+  facet_wrap(~year, scales = "free")+ 
+  scale_color_manual(values = colors_full) +
+  theme_minimal() +
+      labs(x = "Week", y = "Hours Fished", 
+           color = "",
+       title = "Butte Creek Hours Fished")
+
+weekly_hours_fished %>% 
+  filter(stream == "deer creek", year == 2023) |> 
+  ggplot(aes(x = week, y = hours_fished, color = factor(subsite))) +
+  geom_point() + 
+  facet_wrap(~year, scales = "free")+ 
+  scale_color_manual(values = colors_full) +
+  theme_minimal() +
+      labs(x = "Week", y = "Hours Fished", 
+           color = "",
+       title = "Deer Creek Hours Fished")
+
+weekly_hours_fished %>% 
+  filter(stream == "feather river", year == 2022) |> 
+  ggplot(aes(x = week, y = hours_fished, color = factor(subsite))) +
+  geom_point() + 
+  facet_wrap(~year, scales = "free")+ 
+  scale_color_manual(values = colors_full) +
+  theme_minimal() +
+      labs(x = "Week", y = "Hours Fished", 
+           color = "",
+       title = "Feather River Hours Fished")
+
+weekly_hours_fished %>% 
+  filter(stream == "yuba river", year == 2023) |> 
+  ggplot(aes(x = week, y = hours_fished, color = factor(subsite))) +
+  geom_point() + 
+  facet_wrap(~year, scales = "free")+ 
+  scale_color_manual(values = colors_full) +
+  theme_minimal() +
+      labs(x = "Week", y = "Hours Fished", 
+           color = "",
+       title = "Yuba River Hours Fished")
+
+weekly_hours_fished %>% 
+  filter(stream == "sacramento river", year == 2022) |> 
+  ggplot(aes(x = week, y = hours_fished, color = factor(subsite))) +
+  geom_point() + 
+  facet_wrap(~year, scales = "free")+ 
+  scale_color_manual(values = colors_full) +
+  theme_minimal() +
+      labs(x = "Week", y = "Hours Fished", 
+           color = "",
+       title = "Sacramento River Hours Fished")
 
 
 weekly_hours_fished %>% 
```

Comment:
Might be woth noting that this is all years since the last few were filtered to a single year!

Reply:
done